share/scripts/inverse/run_gromacs.sh

```diff
@@ -133,14 +153,15 @@ else
   echo "${0##*/}: No walltime defined, so no time limitation given to $mdrun"
 fi
 
-#>gmx-5.1 has new handling of bonded tables
-if [[ ${mdrun_opts} != *tableb* ]]; then
+#gmx-5.1 has new handling of bonded tables - is it true only for ver.5.1 or above too???
```

Comment:
It is true for all versions >=5.1.3 and 5.1.[12] are broken with bonded tables anyhow, so undo.